insights/terraform/projects/bb2/services/quicksight/modules/quicksight-analysis-prod-applications/main.tf

```diff
@@ -73,7 +73,7 @@ resource "aws_quicksight_analysis" "quicksight_analysis_prod_applications" {
app_fhir_v1_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V1 EOB Requests', {calc_app_fhir_v1_eob_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V1 Coverage Requests', {calc_app_fhir_v1_coverage_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V1 Patient Requests', {calc_app_fhir_v1_patient_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V1 EOB With Since Requests', {calc_app_fhir_v1_eob_since_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V1 Coverage With Since Requests', {calc_app_fhir_v1_coverage_since_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V2 Total Requests', {calc_app_fhir_v2_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V2 EOB Requests', {calc_app_fhir_v2_eob_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V2 Coverage Requests', {calc_app_fhir_v2_coverage_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V2 Patient Requests', {calc_app_fhir_v2_patient_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V2 EOB With Since Requests', {calc_app_fhir_v2_eob_since_call_real_count},\n$${DYNAMICFIELD} = 'FHIR V2 Coverage With Since Requests', {calc_app_fhir_v2_coverage_since_call_real_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Authorize Initial' , {calc_app_authorize_initial_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Medicare Login Redirect OK', {calc_app_medicare_login_redirect_ok_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Medicare Login Redirect FAIL', {calc_app_medicare_login_redirect_fail_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Auth Start OK', {calc_app_authentication_start_ok_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Auth Start FAIL', {calc_app_authentication_start_fail_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Auth Matched New Enrollee Real', {calc_app_authentication_matched_new_bene_real_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Auth Matched New Enrollee Synthetic', {calc_app_authentication_matched_new_bene_synthetic_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Auth Matched Returning Enrollee Real', {calc_app_authentication_matched_returning_bene_real_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Auth Matched Returning Enrollee Synthetic', {calc_app_authentication_matched_returning_bene_synthetic_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW SLS CallBack OK Real', {calc_app_sls_callback_ok_real_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW SLS CallBack OK Synthetic', {calc_app_sls_callback_ok_synthetic_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW SLS CallBack FAIL', {calc_app_sls_callback_fail_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Approval View GET OK Real', {calc_app_approval_view_get_ok_real_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Approval View GET OK Synthetic', {calc_app_approval_view_get_ok_synthetic_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Approval View GET FAIL', {calc_app_approval_view_get_fail_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Approval View POST OK Real', {calc_app_approval_view_post_ok_real_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Approval View POST OK Synthetic', {calc_app_approval_view_post_ok_synthetic_count},\n$${DYNAMICFIELD} = 'AUTH-FLOW Approval View POST FAIL', {calc_app_approval_view_post_fail_count},\n 0)"
```

Comment:
@dtisza1 so that 0 at the very end is what makes the values default to 0. It looks like you removed it and replace it with {calc_app_grant_and_archived_real_bene_deduped_count}. Is that correct? Is the reason to default to this metric if none selected?

Reply:
> @dtisza1 so that 0 at the very end is what makes the values default to 0. It looks like you removed it and replace it with {calc_app_grant_and_archived_real_bene_deduped_count}. Is that correct? Is the reason to default to this metric if none selected?

That's right. I'm not sure why, but the original (non-TF) analysis was defaulting to the 1st value, so that 0 was never an issue. After the terraforming, it is initially using that 0 value instead of the 1st dynamic field. If that makes sense?